### DISCUSSION TITLE
WP30-EU power strip button state

### DISCUSSION THREAD

**Discussion by LeeWiu** [Koenkk/zigbee2mqtt#14088] [Devices]
Hi, I have a power strip WP30-EU, in stable v1.27.2-1 zigbee2mqtt (home assistant addon) if I click any Endpoint: l1-l3 than all 3 relee goes on or off.
If I add to edge zigbee2mqtt, clicking on Endpoint: l1-l3 it's ok, l1 will toggle first relee, l2 the second and l3 the third.
The biggest issue is if I press the buttons on the power strip, (stable or edge z2m) will not change the state in home assistant.
Please advice.

**Comment by paifre** [comment]
Same issue with a power strip received few days ago.
Seems a new version may be on the market with same ref but different behavior
"If I add to edge zigbee2mqtt" --> @TeHashX may you explained what configuration you've done ? I'm not familiar with  Thx

**Comment by LeeWiu** [comment]
Install Z2M Edge https://github.com/Koenkk/zigbee2mqtt/issues/14107#issuecomment-1257136179

**Reply by paifre** [reply]
Many thanks
